src/common/opencl.c

```diff
@@ -2969,10 +2975,13 @@ cl_event *dt_opencl_events_get_slot(const int devid, const char *tag)
     int newevents = *maxevents + DT_OPENCL_EVENTLISTSIZE;
     cl_event *neweventlist = calloc(newevents, sizeof(cl_event));
     dt_opencl_eventtag_t *neweventtags = calloc(newevents, sizeof(dt_opencl_eventtag_t));
+    const size_t allmem = (newevents * (sizeof(cl_event) + sizeof(dt_opencl_eventtag_t))) / 1024lu;
     if(!neweventlist || !neweventtags)
     {
-      free(neweventlist);
-      free(neweventtags);
+      if(neweventlist) free(neweventlist);
+      if(neweventtags) free(neweventtags);
```

Comment:
Where exactly free(NULL) can have bad effects?

Reply:
Again right :-( seems i was too tired / overworked. Will take a free day at the keyboard. remembering wrongly the issue with aligned free ...